### PR TITLE
clarify what we do

### PR DESCRIPTION
**Pull request by ultrasaurus**
Attempting to clarify that this is all volunteer-driven and a little about our mission and stuff.  For context, we've spent quite a bit of time with lawyers recently working on describing our [operational framework](https://docs.google.com/document/d/1fwRpSmL6zexd8SLKe0bFZxk7fZsaavReOHBpDK-8xbE/edit#) and happily ended up right where we are. 

I factored out the Guidelines for *Bridge workshops (so it is on the about page as well as the new event page) and added some intro text:
image

Also added some narrative text to the home page:
image

### CODE REVIEW

**Comment by ultrasaurus** [comment]
@missherico can you review the text?

**Comment by iisa** [comment]
LGTM.
It's Important to clarify the support Bridge Foundry provides plus highlight the successful work of our volunteer community.

File: app/views/events/_guidelines.html.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<h2 class="mb-10-only">Guidelines for *Bridge workshops</h2>
+  <p>
+  <a href="http://bridgefoundry.org/">Bridge Foundry</a>'s mission is to
+  empower people with technology through teaching and facilitating access,
+  enlarging the community of people who give back and teach others.
+  One of the ways we do that is by providing this online registration
```
Comment (tjgrathwell) [inline app/views/events/_guidelines.html.erb]: 'online registration system' sounds cold and is redundant with the next paragraph, maybe remove it?

**Comment by ultrasaurus** [comment]
thanks for the feedback @tjgrathwell  -- will have to think about how to communicate this more warmly.  We're finding that folks new to the whole thing are getting confused when they come in through links from other websites.  For now, this is a good start.

**Pull request merged**